content/contributing/project-information/bevy-organization.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "The Bevy Organization"
+insert_anchor_links = "right"
+[extra]
+weight = 2
++++
+
+The Bevy Organization is the group of people responsible for stewarding the Bevy project. It handles things like merging pull requests, choosing project direction, managing bugs / issues / feature requests, running the Bevy website, controlling access to secrets, defining and enforcing best practices, etc.
+
+:::info
+You **do not** need to be a member of the Bevy Organization to contribute to Bevy. Community contributors (this means you) can freely open issues, submit pull requests, and review pull requests.
```

Comment:
I think this should be worded to be more actively encouraging. Not just that you can / are allowed to, but that you are welcome to.

Reply:
Agreed.